frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -137,13 +137,15 @@ async function getSingleInsight(
     dashboardId: number,
     queryId: string,
     refresh: RefreshType,
-    methodOptions?: ApiMethodOptions
+    methodOptions?: ApiMethodOptions,
+    filters_override?: DashboardFilter
```

Comment:
Let's use camelCase in JS:
```suggestion
    filtersOverride?: DashboardFilter
```